modules/core/ui/map/config/install/farm_map.map_type.locations.yml

```diff
@@ -0,0 +1,11 @@
+langcode: en
+status: true
+dependencies:
+  enforced:
+    module:
+      - farm_ui_map
+id: locations
+label: Locations
+description: 'The farmOS locations map.'
```

Comment:
This is a nice change. It feels a lot cleaner to have a "map type" for a category of assets than to have one for a specific page.

Reply:
I agree that we generally don't need map types for specific pages, but the dashboard is an important exception. See my other comment with ideas to encapsulate this in a reusable `locations` map behavior.